code-challenges.test.js

```diff
@@ -1,5 +1,7 @@
 // ASSESSMENT 3: Coding Practical Questions with Jest
 
+const { describe } = require("yargs")
+
```

Comment:
This line gets added when you install Jest. Just a heads up that sometimes it can cause issues. If you get weird behaviors in your file commenting it out can be helpful.